apkid/rules/dex/obfuscators.yara

```diff
@@ -169,4 +171,18 @@ rule arxan_multidex : obfuscator
     not arxan
 }
 
+rule allatori_demo : obfuscator
+{
+  meta:
+    description = "Allatori demo"
+    url         = "http://www.allatori.com/features.html"
+    author      = "Eduardo Novella"
+    example     = "7f2f5aac9833f7bdccc0b9865f5cc2a9c94ee795a285ef2fa6ff83a34c91827f"
+
+  strings:
+    $s = "ALLATORIxDEMO"
```

Comment:
I thought I commented on this yesterday, but I must have forgot to submit?

This string is really naked. I saw the example code you posted was using this identifier as a method name, so is it possible to make this rule something like `;->ALLATORIxDEMO(`.

Also, are there any other modifications done that we could fingerprint? E.g. string decryption constants, opcodes, etc.